english/meditate/meditate.md

```diff
@@ -1,6 +1,8 @@
 You recently read that meditation helps you feel more awake, relieves stress, and helps you prepare for the day ahead.
 
-So you begin to meditate for 30 minutes and when you finish, you feel more refreshed than ever.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.